dashboard/client/src/pages/node/hook/useNodeList.ts

```diff
@@ -50,34 +51,23 @@ export const useNodeList = () => {
     };
   }, [getList]);
 
-  const finalSortFunc = (a: NodeDetail, b: NodeDetail) => {
-    const sortFuncs: ((a: NodeDetail, b: NodeDetail) => number)[] = [
-      // user override first
-      sorterFunc,
-      // Head node is always first
-      (a, b) => (a.raylet.isHeadNode ? 0 : 1) - (b.raylet.isHeadNode ? 0 : 1),
-      // Then sort by state
-      (a, b) => (a.raylet.state > b.raylet.state ? 1 : -1),
-      // Finally sort by nodeId
-      (a, b) => (a.raylet.nodeId > b.raylet.nodeId ? 1 : -1),
-    ];
+  const nodeListWithState = nodeList
+    .map((e) => ({
+      ...e,
+      state: e.raylet.state,
+    }))
+    .sort(sorterFunc);
 
-    for (const sortFunc of sortFuncs) {
-      const val = sortFunc(a, b);
-      if (val !== 0) {
-        return val;
-      }
-    }
-    return 0;
-  };
+  const sortedList = _.sortBy(nodeListWithState, [
+    (obj) => !obj.raylet.isHeadNode,
+    (obj) => obj.raylet.state,
```

Comment:
Is there any way not to rely on that "alive" alphabetically comes before "dead"?